# Convert BSD devices to Linux.
# mbuf, pf, proc, and sensor are not available in Linux.

--- symon/symon/symon.conf.orig	2008-03-18 03:46:17.173140738 +0000
+++ symon/symon/symon.conf	2008-03-18 04:49:07.463331989 +0000
@@ -3,13 +3,14 @@
 #
 # Demo configuration for symon. See symon(8) for BNF.
 
-monitor { cpu(0),  mem,
-	  if(lo0),
-#	  pf,
-#         mbuf,
-#         sensor(0),
-#         proc(httpd),
-#         if(xl0), if(de0), if(wi0),
-#	  io(wd1), io(wd2), io(wd3), io(cd0)
-	  io(wd0)
+monitor {
+	cpu(0),
+	cpu(1),
+#	cpu(2),
+	mem,
+	if(lo),
+	if(eth0),
+#	if(eth1),
+#	io(hda), io(hdb), io(hdc), io(hdd),
+	io(sda), io(sdb)
 } stream to 127.0.0.1 2100
